.github/workflows/check_version.sh

```diff
@@ -1,12 +1,27 @@
 #!/bin/sh
 
 version=$1
+path=$2
 
-packageversion=$(cat $2/package.json | grep -Eo '\"version\": \"[[:digit:]]+\.[[:digit:]+\.[[:digit:]]+\"')
-formattedversion="\"version\": \"$version\""
-
-if [ "$formattedversion" = "$packageversion" ]; then
-  exit 0
+if [ "${path: -5}" = .json ]; then
```

Comment:
I think you don't need path: -5 easily replaceable with 
```if [[ $path == *.json ]]; then``` same in go IMHO

Reply:
Afaik `if [[ ... ]]` is only present in bash, and we like to use just sh.

Reply:
Summon @nandor-magyar

Reply:
You can't do string extrapolation you are using in a POSIX shell. Use cut instead.

Reply:
@nandor-magyar which POSIX compatibility checker, did you used before?